mmv1/products/beyondcorp/SecurityGatewayApplication.yaml

```diff
@@ -141,6 +152,106 @@ properties:
+        - name: external
+          type: NestedObject
+          description: List of the external endpoints to forward traffic to.
+          properties:
+            - name: endpoints
+              type: Array
+              description: List of the endpoints to forward traffic to.
+              required: true
+              item_type:
+                type: NestedObject
+                properties:
+                  - name: hostname
+                    type: String
+                    description: Hostname of the endpoint.
+                    required: true
+                  - name: port
+                    type: Integer
+                    description: Port of the endpoint.
+                    required: true
+        - name: proxyProtocol
+          type: NestedObject
+          description: Shared proxy configuration for all apps.
+          properties:
+            - name: allowedClientHeaders
+              type: Array
+              description: The configuration for the proxy.
+              item_type:
+                type: string
+            - name: contextualHeaders
+              type: NestedObject
+              description: Configuration for the contextual headers.
+              properties:
+                - name: userInfo
+                  type: NestedObject
+                  description: User info configuration.
+                  properties:
+                    - name: outputType
+                      type: Enum
+                      description: The output type of the delegated user info.
+                      enum_values:
+                        - 'OUTPUT_TYPE_UNSPECIFIED'
+                        - 'PROTOBUF'
+                        - 'JSON'
+                        - 'NONE'
+                - name: groupInfo
+                  type: NestedObject
+                  description: Group info configuration.
+                  properties:
+                    - name: outputType
+                      type: Enum
+                      description: The output type of the delegated group info.
+                      enum_values:
+                        - 'OUTPUT_TYPE_UNSPECIFIED'
+                        - 'PROTOBUF'
+                        - 'JSON'
+                        - 'NONE'
+                - name: deviceInfo
+                  type: NestedObject
+                  description: Device info configuration.
+                  properties:
+                    - name: outputType
+                      type: Enum
+                      description: The output type of the delegated device info.
+                      enum_values:
+                        - 'OUTPUT_TYPE_UNSPECIFIED'
+                        - 'PROTOBUF'
+                        - 'JSON'
+                        - 'NONE'
+                - name: outputType
+                  type: Enum
+                  description: Default output type for all enabled headers.
+                  enum_values:
+                    - 'OUTPUT_TYPE_UNSPECIFIED'
```

Comment:
Can we remove the _UNSPECIFIED type in all of these? It's usually not a valid type to specify in Terraform, so it doesn't make sense to include here & in documentation